install/cupy_builder/cupy_setup_build.py

```diff
@@ -575,30 +575,34 @@ def _nvcc_gencode_options(cuda_version):
                          ('compute_75', 'sm_75'),
                          ('compute_80', 'sm_80'),
                          ('compute_86', 'sm_86'),
-                         'compute_86']
-        elif cuda_version >= 11000:
+                         ('compute_87', 'sm_87'),
+                         'compute_87']
+        elif cuda_version >= 11010:
             arch_list = ['compute_35',
                          'compute_50',
                          ('compute_60', 'sm_60'),
                          ('compute_61', 'sm_61'),
                          ('compute_70', 'sm_70'),
                          ('compute_75', 'sm_75'),
                          ('compute_80', 'sm_80'),
-                         'compute_80']
-        elif cuda_version >= 10000:
-            arch_list = ['compute_30',
+                         ('compute_86', 'sm_86'),
+                         'compute_86']
+        elif cuda_version >= 11000:
+            arch_list = ['compute_35',
                          'compute_50',
                          ('compute_60', 'sm_60'),
                          ('compute_61', 'sm_61'),
                          ('compute_70', 'sm_70'),
                          ('compute_75', 'sm_75'),
-                         'compute_70']
-        elif cuda_version >= 9020:
+                         ('compute_80', 'sm_80'),
+                         'compute_80']
+        elif cuda_version >= 10000:
```

Comment:
```suggestion
        elif cuda_version >= 10020:
```

c.f. https://docs.nvidia.com/cuda/archive/10.2/cuda-compiler-driver-nvcc/index.html#virtual-architecture-feature-list